numpy/core/tests/test_indexing.py

```diff
@@ -547,6 +547,17 @@ def test_object_assign(self):
         b[[0], :3] = [[1, (1,2), 3]]
         assert_array_equal(a, b[0])
 
+        # Check that swapping of axes works
+        b = b.T
+        b[:3, [0]] = [[1], [(1,2)], [3]]
+        assert_array_equal(a, b[:, 0])
+
+        # Another test for the memory order of the subspace
+        # (though it should not matter)
```

Comment:
Yeah, it just tests that, though actually maybe I could also test if the result is actually correct anyway. But how would you explictly test it?